### ISSUE TITLE
Ch. 4 ERROR: missing data for column "state_us_abbreviation"

### ISSUE THREAD

**Issue by chouci** [anthonydb/practical-sql#8]
Listing 4.3 query:

COPY us_counties_2010
FROM '/Users/Hipolito/Downloads/us_counties_2010.csv'
WITH (FORMAT CSV, HEADER);

error:

ERROR:  missing data for column "state_us_abbreviation"
CONTEXT:  COPY us_counties_2010, line 2: ""
SQL state: 22P04

Mac OS, installed PostgreSQL exactly as suggested in the book, ran the query using pgAdmin4.

**Comment by anthonydb** [comment]
Looks like you solved this? Just making sure. If so, what was the issue?

**Comment by chouci** [comment]
Original problem:
Originally, I downloaded from https://github.com/anthonydb/practical-sql/tree/master/Chapter_04 the file us_counties_2010.csv, but, in my haste, instead of saving the contents as csv, I saved the _link_ as .txt (so I downloaded the web page as text, instead of the .csv file). That's why the error message indicated missed data. 

Solution:  
Clone data directly to my machine using GitHub.
